lighthouse-core/audits/metrics/largest-contentful-paint.js

```diff
@@ -27,21 +27,32 @@ class LargestContentfulPaint extends Audit {
       title: str_(i18n.UIStrings.largestContentfulPaintMetric),
       description: str_(UIStrings.description),
       scoreDisplayMode: Audit.SCORING_MODES.NUMERIC,
-      requiredArtifacts: ['traces', 'devtoolsLogs'],
+      requiredArtifacts: ['traces', 'devtoolsLogs', 'TestedAsMobileDevice'],
     };
   }
 
   /**
-   * @return {LH.Audit.ScoreOptions}
+   * @return {{mobile: {scoring: LH.Audit.ScoreOptions}, desktop: {scoring: LH.Audit.ScoreOptions}}}
    */
   static get defaultOptions() {
     return {
-      // 25th and 13th percentiles HTTPArchive -> median and p10 points.
-      // https://bigquery.cloud.google.com/table/httparchive:lighthouse.2020_02_01_mobile?pli=1
-      // https://web.dev/lcp/#what-is-a-good-lcp-score
-      // see https://www.desmos.com/calculator/1etesp32kt
-      p10: 2500,
-      median: 4000,
+      mobile: {
+        // 25th and 13th percentiles HTTPArchive -> median and p10 points.
```

Comment:
why is this 13?

is this the x percentile of all web traffic or just desktop? (same question for mobile control points). comment doesn't make it clear, but the link to bigquery specifies mobile so I think it's filtering device type?

Reply:
> why is this 13?

#10715 changed the second control point from the PODR to p10, but in order to not change the curves, the p10 point ended up at a somewhat arbitrary percentile for each audit.

(this PR isn't changing any of the mobile numbers, though ([?w=1](https://github.com/GoogleChrome/lighthouse/pull/10756/files?w=1) makes this a little clearer))

> is this the x percentile of all web traffic or just desktop? (same question for mobile control points). comment doesn't make it clear, but the link to bigquery specifies mobile so I think it's filtering device type?

Right, the mobile and desktop data come from the percentiles in the respective http archive tables (`*_mobile` and `*_desktop`) mentioned in each comment.

Reply:
I think this was decided to leave the LCP points as they were at some point prior and this is just a noop whitespace change.